hole/play.go

```diff
@@ -496,6 +496,12 @@ func play(
 		stdoutBytes = bytes.ReplaceAll(stdoutBytes, []byte("\x00"), []byte("\n"))
 	}
 
+	// INTHEFIX Why on earth does Vyxal print `False` on every run?????????????
```

Comment:
@lyxal

Why does this happen, and how do I stop it? 🤔

Reply:
Are you sure Vyxal 3 isn't good to go? Since you insisted that Vyxal 2 is better, I respected that. However, if the very very latest is good to go, I have that one ready and lurking as well.

Reply:
I've fixed the printing false issue - there was a leftover debugging print statement that made it into the release 😅

Vyxal 2.22.1 contains the fix

Reply:
Vyxal 2 is better suited for addition at this time, as we're working on an improved version of Vyxal 3 (3.5). Until then, v3 isn't  something that's ready to add

Reply:
> I've fixed the printing false issue - there was a leftover debugging print statement that made it into the release 😅
> 
> Vyxal 2.22.1 contains the fix

Awesome stuff, thanks! Rebuilding now.